torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1789,13 +1784,6 @@ at::Tensor AtenXlaType::masked_select(const at::Tensor& self,
                                       const at::Tensor& mask) {
   XLA_FN_COUNTER("xla::");
   XLATensor self_tensor = bridge::GetXlaTensor(self);
-  // Initially make XLA handled masked_select() handling experimental, and
-  // opt-in. Only the XLA TPU backend for now implements the dynamic dimension
-  // setting required by the masked_select implementation.
-  if (!DebugUtil::ExperimentEnabled("masked_select") ||
```

Comment:
I'd still keep the experimental checks for now.

Reply:
Done